src/run_reps.jl

```diff
@@ -0,0 +1,134 @@
+using Dates
+using Random
+
+"""
+    run_rme(rme_path::String, n_threads::Int64, reps::Int64, result_path::String; start_year::Int64=2022, end_year::Int64=2099, batch_size::Int64=10, start_batch::Int64=1, RCP_scen::String="SSP 2.45", gcm::String="CNRM_ESM2_1", rnd_seed::Int64=1234)::Nothing
+
+Run counterfactual scenarios with ReefModEngine.jl and save result set to desired dir.
+
+# Arguments
+- `rme_path` : Path to REM folder.
```

Comment:
If we move the RME option setting step outside this function as suggested below, we can remove this (and other) argument lines, but if we decide to keep it:


```suggestion
- `rme_path` : Path to RME folder.
```